src/result.js

```diff
@@ -1305,14 +1305,14 @@ var Result = CreateClass({
 
                         var supplementalDamageInfo = [];
                         const supplementalInfo = supplemental.collectSkillInfo(skilldata.supplementalDamageArray, {remainHP: m.data[key].remainHP});
-                        if ("keys" in supplementalInfo && "values" in supplementalInfo && "total" in supplementalInfo) {
+                        if ("keys" in supplementalInfo && "values" in supplementalInfo && "total" in supplementalInfo && supplementalInfo.total > 0) {
                             supplementalDamageInfo.push(
                                 <table key={key + "-supplementalDamageTable"} className="table table-bordered" style={{"marginBottom": "0px"}} >
                                     <thead>
                                         <tr>
                                             <th className="bg-success" style={{"fontSize": "10pt"}}>{intl.translate("与ダメージ上昇効果のソース", locale)}</th>
                                             {supplementalInfo.keys.map( function (v, ind) {
-                                                return <th key={ind} className="bg-success" style={{"fontSize": "10pt"}}>{intl.translate(v, locale)}</th>
+                                                return <th key={ind} className="bg-success" style={{"fontSize": "10pt"}}>{intl.translate(v[0], locale) + intl.translate(("supplemental_"+v[1]), locale)}</th>
```

Comment:
(v, ind) -> ([key, type], ind)

Reply:
Reason: 
v[0] and v[1] is hard coded magic number, Readers need to search what was the values. 
[key, type] naming info can give the hints.